src/server/socketHandlers/parseChannel.js

```diff
@@ -0,0 +1,9 @@
+export default function parseChannel(channelName) {
```

Comment:
channels are written in standard REST format eg `presence/:meetingId/:randomVarId`